app/src/main/java/ch/epfl/sdp/drone3d/drone/api/DroneData.kt

```diff
@@ -104,4 +104,9 @@ interface DroneData {
      * will not be used anymore.
```

Comment:
Why should we use liveData here ? I'm not sure to understand the documentation either.